app/views/submissions/repo_browser.html.erb

```diff
@@ -62,6 +62,11 @@
         (<span id='current_revision_timestamp_display'>
            <%= l(@revision_timestamp, format: :long_date) %>
         </span>)
+        <br>
+        <%= t('browse_submissions.currently_collected') %> 
+        <%= @grouping.is_collected ? "#" + raw(@grouping.submissions.last.revision_number) + 
```

Comment:
<code>@grouping.submissions.last.revision_timestamp</code> returns incorrect time which is why I had to use <code>@revisions_history</code>.